### PR TITLE
Implement Poloniex Ticker

### PR DESCRIPTION
**Pull request by tmlee**

### CODE REVIEW

**Comment by Swingcloud** [comment]
maybe close PR and try again, it will fix anyway @tmlee

**Comment by tmlee** [comment]
@Swingcloud Can retry the build directly on the CI if it fails

**Comment by Swingcloud** [comment]
@tmlee I see! next time I will just click restart build instead close all PR

File: lib/cryptoexchange/exchanges/poloniex/models/market_pair.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Cryptoexchange::Exchanges
+  module Poloniex
+    module Models
+      class MarketPair < Cryptoexchange::Models::MarketPair
```
Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/poloniex/models/market_pair.rb]: @tmlee please do not create child model classes from now on, thanks.

Comment (tmlee) [inline lib/cryptoexchange/exchanges/poloniex/models/market_pair.rb]: Alright got rid of that @dominicwong617

**Pull request merged**